kong.conf.default

```diff
@@ -1129,6 +1129,31 @@
                                  # Detailed discussion of this behavior is
                                  # available in the online documentation.
 
+#pg_keepalive_timeout =          # Specify the maximal idle timeout (in ms)
+                                 # for the postgres connections in the pool.
+                                 # If values is set to 0 then the timeout interval
+                                 # is unlimited.
+                                 #
+                                 # If not specified this value will be same as `lua_socket_keepalive_timeout`
+
+#pg_pool_size =                  # Specifies the size limit (in terms of connection count) for
+                                 # postgres server.
+                                 # Note that this connection pool is per nginx worker rather than per kong instance,
+                                 # so size limit specified applies to every single Nginx worker process.
+                                 #
+                                 # If not specified default value is same as `lua_socket_pool_size`
```

Comment:
```suggestion
#pg_pool_size =                  # Specifies the size limit (in terms of connection 
                                 # count) for the Postgres server.
                                 # Note that this connection pool is intended 
                                 # per Nginx worker rather than per Kong instance.
                                 #
                                 # If not specified, the default value is the same as `lua_socket_pool_size`
```
description / format